test/e2e/malformed_resources_test.go

```diff
@@ -19,22 +18,12 @@ type MalformedResourcesSuite struct {
 }
 
 func (s *MalformedResourcesSuite) TestMalformedWorkflow() {
-	s.Given().
-		Exec("kubectl", []string{"apply", "-f", "testdata/malformed/malformed-workflow.yaml"}, fixtures.NoError).
-		WorkflowName("malformed").
-		When().
-		// it is not possible to wait for this to finish, because it is malformed
-		Wait(3 * time.Second).
-		Then().
-		ExpectWorkflow(func(t *testing.T, metadata *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
-			assert.Equal(t, "malformed", metadata.Name)
-			assert.Equal(t, wfv1.WorkflowFailed, status.Phase)
-		})
+	s.Given().Exec("kubectl", []string{"apply", "-f", "testdata/malformed/malformed-workflow.yaml"}, fixtures.ErrorOutput("unknown field \"spec.arguments.parameters.someParam\""))
```

Comment:
Had to rewrite the tests in this file because malformed workflows will now be rejected at the API level instead of at runtime. We could test the runtime behavior of malformed workflows by adding a new profile that uses the minimal CRDs and having `test-executor` use that, but that seems overkill.